demos/smart_classroom_demo/create_list.py

```diff
@@ -41,4 +38,4 @@
         objects[label] = [path]
 
 with open('faces_gallery.json', 'w') as outfile:
-    json.dump(objects, outfile, indent=4)
+    json.dump(objects, outfile, indent=4)
```

Comment:
"No newline". Also, it seems this file doesn't need to be modified at all. Please revert

Reply:
Script from omz demo (Smart_classroom_demo).